paimon-spark/paimon-spark-common/src/main/scala/org/apache/paimon/spark/PaimonBaseScan.scala

```diff
@@ -141,37 +160,14 @@ abstract class PaimonBaseScan(
       ""
     }
 
-    val latestSnapshotId = if (table.latestSnapshot().isPresent) {
-      Some(table.latestSnapshot().get.id)
-    } else {
-      None
-    }
-
-    val latestSnapshotIdStr = if (latestSnapshotId.isDefined) {
-      s", LatestSnapshotId: [${latestSnapshotId.get}]"
+    val latestSnapshotIdStr = if (latestSnapshot.nonEmpty) {
```

Comment:
Why we need to get latest snapshot? Why not just remove it.

Reply:
It's not very useful and will increase request, I'm inclined to remove it.